.github/actions/build-setup/action.yml

```diff
@@ -43,18 +44,19 @@ runs:
 
         # prepare move lang prover tooling.
         # By setting these values the dev-setup.sh script can detect already installed executables (and check versions).
-        echo 'Z3_EXE='${HOME}/bin/z3 | tee -a $GITHUB_ENV
-        echo 'CVC4_EXE='${HOME}/bin/cvc4 | tee -a $GITHUB_ENV
-        echo 'DOTNET_ROOT='${HOME}/.dotnet/ | tee -a $GITHUB_ENV
-        echo 'BOOGIE_EXE='${HOME}/.dotnet/tools/boogie | tee -a $GITHUB_ENV
+        echo 'Z3_EXE='/opt/bin/z3 | tee -a $GITHUB_ENV
```

Comment:
Dev setup normally wouldn't use -n and hence would install most tooling in the home directory instead of /opt.   Introduced -n to avoid the home dir since it would be effectively wiped in the docker images in github actions.  Dotnet defaults to ~/.dotnet, rust to ~/.rustup and ~/.cargo, etc, and the once off tools we grab from git would go to ~/bin - we have a .profile file section of the dev-setup.sh to build output for local devs, again unfortunately wiped in CI/CD by gha.  Also that file is meant to be sourced, so it has 'export ' in front of each var.     The cat could be a good change - perhaps less error prone, or an all-or-nothing error on a bad edit.